assets/cubyz/blocks/basalt.zig.zon

```diff
@@ -1,17 +1,23 @@
 .{
 	.tags = .{.stone},
-	.blockHealth = 20,
+	.blockHealth = 30,
 	.drops = .{
 		.{.items = .{.auto}},
 	},
 	.item = .{
 		.material = .{
+		.modifiers = .{
+			.{
+				.id = "fragile",
+				.strength = 0.20,
+			},
+		},
```

Comment:
Please fix the indentation of your modifiers.
Also in my opinion the modifiers should come after all the other properties.
(should be done in all files)